library/std/src/sync/poison/mutex.rs

```diff
@@ -181,10 +181,26 @@ pub struct Mutex<T: ?Sized> {
     data: UnsafeCell<T>,
 }
 
-// these are the only places where `T: Send` matters; all other
-// functionality works fine on a single thread.
+/// Mutex is a container that wraps `T`, so it's necessary for `T` to be `Send`
+/// to safely send `Mutex` to another thread. This ensures that the protected
+/// data can be accessed safely from multiple threads without causing data races
+/// or other unsafe behavior.
```

Comment:
The part about multiple threads is relevant for `Sync`. If you rely on `Send`, you have exclusive access anyways.